tutorials/graph-hello-world/graph-hello-world.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Hello Graph! Write Your First SAP Graph Application
+description: Get a brief introduction to SAP Graph, including the SAP Graph Navigator and programming basics.
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, software-product>sap-business-technology-platform]
+primary_tag: software-product>sap-integration-suite
+---
+
+## Details
+### You will learn
+  - What SAP Graph entails
+  - How to navigate connected data entities in the SAP Graph Navigator
+  - `hello world` programming essentials to try out SAP Graph
+
+  SAP Graph is a new unified API for SAP. It addresses run-away API complexity and integration challenges by providing developers a single connected and unified view of all their business data, consolidating the data models of data sources like SAP S/4HANA, SAP Sales Cloud and SAP SuccessFactors into one, unified and connected, data model, representing all the data in a landscape. We call this the Business Data Graph.
+
+  !![SAP Graph Overview](SAP-Graph-overview_sm.png)
+
+  A data graph is a connected graph, with data entities serving as the nodes of the graph and edges that represent relationships (which we call associations) between those entities. The nodes of the Business Data Graph are the common enterprise business objects, like Customer, Supplier, Product, Sales and Purchase Order, General Ledger, etc.
+
+SAP Graph was productively released for early adopters in December of 2021. We offer dedicated support for early adopters of this innovative solution. [Sign up to be an early adopter] (https://influence.sap.com/sap/ino/#campaign/2940) of SAP Graph or contact us at sap.graph@sap.com if you are interested.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Explore SAP Graph on the SAP Graph Navigator)]
+
+The SAP Graph API is documented in the new [SAP Graph Navigator](https://navigator.graph.sap/entities). Go ahead and follow the link to the SAP Graph Navigator application to explore the API. The index, on the left of the application, shows you over 1,500 entities grouped by data source. The first group of entities are the unified entities. Here is an example of a unified entity, `sap.graph/WorkAssignment`.
+
+!![WorkAssignment unified entity in SAP Graph Navigator](blog1_navigator_sm.png)
+
+In this tutorial, you will create a small application that fetches the unified entity `SalesQuote`. Use the SAP Graph Navigator to look up the definition, and corresponding attributes for `SalesQuote`.
+
+[DONE]
+
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Prepare the Node Development Environment)]
+
+We will use **Node.js** and **npm** to build this application, and to keep it extremely simple, we will use the **Express** web framework. Not familiar with Node.js and Express? We recommend this article: <https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction>. That article also explains how to easily set up a Node development environment, whether you use Windows, MacOS or Linux.
```

Comment:
Use You and not the royal We.(unless, in rare cases, you are referring to SAP).

Reply:
Rephrased.